### DISCUSSION TITLE
How to get the URL of the current tab?

### DISCUSSION THREAD

**Discussion by K0-RR** [pyscript/pyscript#1865] [Q&A]
I'm trying to make a Chrome extension using Pyscript instead of JS. Is it possible to get the address of the page that is open?

**Comment by WebReflection** [comment]
```python
from pyscript import window

print(window.location.href)
```

that should do, although it's very likely that it returns the background page or the extension URL and, if that's not what you are after, there are extension APIs to do that, and you still can reach those APIs via the `window` or the `js` import (maybe for extension APIs the `js` is better so ... `import js` and that's it).

**Comment by WebReflection** [comment]
No feedback for a week, I think we can close this.
